### DISCUSSION TITLE
I want to change status from table row.

### DISCUSSION THREAD

**Discussion by erenergul** [filamentphp/filament#4720] [Help]
Is there any way to change status directly from Table. If possible multiple column will be better. Select 5 items and change status from new to "confirmed" for example. I dont want to go edit page and change status each one. How can i achieve it ? I couldn't find any example. I will be very grateful. Thank you

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/tables/columns/select

**Reply by erenergul** [reply]
Hello Dan , Thank you for your reply. Is there any way to do that with bulkaction multiple changes ? Like we delete 10 items. Is there any chance to confirmed 10 items ?
